web/concrete/src/Console/Command/GenerateIDESymbolsCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Concrete\Core\Console\Command;
+
+use Concrete\Core\Support\Symbol\ClassSymbol\ClassSymbol;
+use Concrete\Core\Support\Symbol\ClassSymbol\MethodSymbol\MethodSymbol;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Exception;
+
+class GenerateIDESymbolsCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('c5:ide-symbols')
+            ->setDescription('Generate IDE symbols')
+            ->addArgument('generate-what', InputArgument::IS_ARRAY, 'Elements to generate [all|ide-classes|phpstorm]', array('all'))
+        ;
+    }
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $rc = 0;
+        try {
+            $what = $input->getArgument('generate-what');
+            if (in_array('all', $what) || in_array('ide-classes', $what)) {
+                $output->write('Generating fake PHP classes to help IDE... ');
+                $this->generateIDEClasses();
+                $output->writeln('<info>done.</info>');
+            }
+            if (in_array('all', $what) || in_array('phpstorm', $what)) {
+                $output->write('Generating PHP metadata for PHPStorm... ');
+                $this->generatePHPStorm();
+                $output->writeln('<info>done.</info>');
+            }
+        } catch (Exception $x) {
+            $output->writeln('<error>'.$x->getMessage().'</error>');
+            $rc = 1;
+        }
+
+        return $rc;
+    }
+    protected function generateIDEClasses()
```

Comment:
These method names look to be reversed, IDE Classes should be the `SymbolGenerator` and PHPStorm should be the `.phpstorm.meta.php` no? Also can you try to keep empty lines between your method declarations as we have in other files (even though it isn't officially required by PSR-1 or 2)